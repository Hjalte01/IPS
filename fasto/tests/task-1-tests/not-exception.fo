fun int main() =
    write(not 'a')
