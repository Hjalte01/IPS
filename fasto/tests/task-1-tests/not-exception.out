Interpreter error: Bad type for operand of not: expected bool, but got char ('a') at line 2, column 15
